packages/lesswrong/components/common/CalendarDate.tsx

```diff
@@ -1,16 +1,20 @@
-import { registerComponent } from '../../lib/vulcan-lib';
+import { Components, registerComponent } from '../../lib/vulcan-lib';
 import React from 'react';
 import moment from '../../lib/moment-timezone';
 import { useTimezone } from '../common/withTimezone';
 
-/// A date rendered with moment().calendar(). Includes a plethora of special
-/// cases like "Yesterday at 1:00 PM", "Last Tuesday at 1:00 PM". Turns into a
-/// more normal date (in locale format) if more than a week away.
+/**
+ * A date rendered with moment().calendar(). Includes a plethora of special
+ * cases like "Yesterday at 1:00 PM", "Last Tuesday at 1:00 PM". Turns into a
+ * more normal date (in locale format) if more than a week away.
+ */
 const CalendarDate = ({date}: {
   date: Date,
 }) => {
+  const { TimeTag } = Components;
+
   const { timezone } = useTimezone();
-  return <span>{moment(new Date(date)).tz(timezone).calendar()}</span>
+  return <TimeTag dateTime={date}>{moment(new Date(date)).tz(timezone).calendar()}</TimeTag>
```

Comment:
The reason is that `date` is actually of type `Date | string`, because date fields are returned as strings in graphql. This is a pretty deeply baked type issue throughout the codebase

Reply:
I have fixed this one case